### PR TITLE
style(core): Found and replaced with closest colors for variables in color.less 

### PR DESCRIPTION
**Pull request by archana-s**
Used nearest-color to find closest colors for colors in colors.less and replaced them throughout core directory. Affects colors on multiple pages.

### CODE REVIEW

**Comment by archana-s** [comment]
Working on fixing a small problem.

File: app/scripts/modules/core/src/entityTag/notifications/notifications.less [hunk 1]
```diff
@@ -1,63 +1,70 @@
 @import (reference) "~core/presentation/less/imports/commonImports.less";
```
Comment (archana-s) [inline app/scripts/modules/core/src/entityTag/notifications/notifications.less]: @jrsquared PTAL

**Comment by archana-s** [comment]
Looks good to me.

**Comment by anotherchrisberry** [comment]
Let's find out!

**Pull request merged**